src/app/board/components/create-column-modal/create-column-modal.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component } from '@angular/core';
+import { MatDialogRef } from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-create-column-modal',
+  templateUrl: './create-column-modal.component.html',
+  styleUrls: ['./create-column-modal.component.scss'],
+})
+export class CreateColumnModalComponent {
+  columnTitle: string = '';
+
+
+  constructor(private dialogRef: MatDialogRef<CreateColumnModalComponent>) {}
+
+
+  save() {
```

Comment:
a good thing to write private/public for each field and method. And returned value type.